react/components/AuthFormContainer/index.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { fontSize } from 'styled-system';
+
+import ArenaMark from 'react/components/UI/ArenaMark/index.svg';
+import { Input } from 'react/components/UI/GenericInput';
+import GenericButton from 'react/components/UI/GenericButton';
+import { preset } from 'react/styles/functions';
+import Text from 'react/components/UI/Text';
+
+const Container = styled.form`
+  max-width: 25em;
+  margin: 0 auto;
+`;
+
+const AuthInput = styled(Input)`
+  margin: 1.5em 0;
+`;
+
+const Button = styled(GenericButton).attrs({ f: 2, type: 'submit' })`
+  margin: 0 auto;
+  padding-left: 2em;
+  padding-right: 2em;
+  color: ${x => x.theme.colors.gray.base};
```

Comment:
You can set all this stuff using the system attrs

Reply:
Hm, how do you mean?

Reply:
```
color: 'gray.base',
p: 6,
```

etc...

Don't make up your own values for things; use the scales we have: https://dzucconi.github.io/ervell/?selectedKind=Styles&selectedStory=Space&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
https://dzucconi.github.io/ervell/?selectedKind=Button&selectedStory=GenericButton%20-%20colors&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
Ah you're talking about the padding values, not color, right?

Reply:
copy paste that snippet into your attrs declaration

Reply:
You don't need a function to set the color here.

Reply:
it'd probably be helpful to publish your storybook